disk/disk_linux.go

```diff
@@ -283,6 +283,14 @@ func PartitionsWithContext(ctx context.Context, all bool) ([]PartitionStat, erro
 			mountPoint := fields[4]
 			mountOpts := fields[5]
 
+			if rootDir := fields[3]; rootDir != "" && rootDir != "/" {
+				if len(mountOpts) == 0 {
```

Comment:
👍 mountOpts is just a string, so this is fastest way (mountOpts should be a list, not string, though).